ledger-core/virtual/execute/execute.go

```diff
@@ -264,7 +263,11 @@ func (s *SMExecute) stepWaitObjectReady(ctx smachine.ExecutionContext) smachine.
 		}))
 		return ctx.Jump(s.stepSendCallResult)
 	case object.Inactive:
-		panic(throw.Impossible())
+		s.prepareExecutionError(throw.E("attempt to call method on object that is completely deactivated", struct {
```

Comment:
this is not correct place to check

Reply:
It's kind of short path for checking. There is one more in 'stepStartRequestProcessing'

Reply:
duplicate request of method after deactivation in previous pulse. I wrote about this in slack.

Reply:
move check to stepStartRequestProcessing